lib/commands/init.js

```diff
@@ -125,7 +133,7 @@ module.exports = Command.extend({
     this.ui.writeLine('');
     const command = `cd ${projectName}`;
     this.ui.writeLine(`  ${chalk.gray('$')} ${chalk.cyan(command)}`);
-    const packageManager = commandOptions.yarn ? 'yarn' : 'npm';
+    const packageManager = commandOptions.pnpm ? 'pnpm' : commandOptions.yarn ? 'yarn' : 'npm';
```

Comment:
Request to simplify the ternaries again 🙏

Reply:
done :tada: